### PR TITLE
Update readme for better understanding

### PR DESCRIPTION
**Pull request by jonathansmirnoff**
## What

- Improved the documentation
- The readme was divided in two sections: the first section it is explained how to use ganache with this truffle box; the other it is explained how to use the box in RSK Network. Also I added a section that explain how to get rbtc/trbtc. 

## Why

- Truffle asked to do this. They wanted to divided the readme in two parts: Ganache and RSK network.

### CODE REVIEW

**Comment by solangegueiros** [comment]
It still do not have smart contracts....
For a smart contract developer is ok, but for a developer who are starting now, it will be very good it he has a example ant contracts and migrations

File: README.md [hunk 1]
```diff
@@ -2,59 +2,80 @@
...
-```
+3. Run the development console.
 
-## Setup environment
+    ```
+    truffle develop
+    ```
 
-The only thing you'll need to do it's to copy your mnemonic to truffle-config.js
+4. Compile and migrate the smart contracts. Note inside the development console we don't preface commands with truffle.
```
Comment (solangegueiros) [inline README.md]: Here, you are talking about smart contracts, but it do not have one...

File: README.md [hunk 2]
```diff
@@ -2,59 +2,80 @@
...
-```bash
-# Console for Mainnet
-truffle console --network mainnet
+### Setup the gas price
+You should check the gas price that you have to use in order send a transactions for any RSK Network. RSK has the property minimum gas price, which tells the minimum gas price that you have to use to pay for a transaction.
 
-# Console forn Testnet
-truffle console --network testnet
-```
+To get the stats for Mainnet go [here](https://stats.rsk.co/) and Testnet go [here](https://stats.testnet.rsk.co/)
```
Comment (solangegueiros) [inline README.md]: So, to check the gas price do I need to go here?
I think you can explain this...

File: README.md (hunk 1 above)
Comment (jonathansmirnoff) [reply README.md]: Yes, you're right. You can think this an empty template. Maybe we can add an example contract in the next iteration.

File: README.md (hunk 2 above)
Comment (jonathansmirnoff) [reply README.md]: To get the minimum gas price, you can get the last block and check that property. I thought that maybe it is easier to understand with the stats.
What do you think it is the best way to explain this?

Comment (jonathansmirnoff) [reply README.md]: I think it is better to link to this page: https://developers.rsk.co/rsk/rbtc/gas/ and explain that also the have the possibility to check the value with the stats

File: README.md (hunk 1 above)
Comment (solangegueiros) [reply README.md]: For my is the simple storage

pragma solidity >=0.4.0 <0.7.0;

contract Storage {
    string public storedData;

    function set(string memory x) public {
        storedData = x;
    }

    function get() public view returns (string memory) {
        return storedData;
    }
}

Comment (jonathansmirnoff) [reply README.md]: we can add a test for it, right?

File: README.md (hunk 2 above)
Comment (solangegueiros) [reply README.md]: Yes, https://developers.rsk.co/rsk/rbtc/gas/  have a good explanation, but it is not practical.
I think explain a little more about how I got the gas fee using the stats page, like:

To know the gas price, go to the stats page at ....
and then you will find it at left. But the value is in wei, and must be converted to hexadecimal
(something like this)

I can not do copy / paste from the stats page :(

File: README.md (hunk 1 above)
Comment (solangegueiros) [reply README.md]: Yes, we need to show how use it at truffle console.
And the most important thing is to have a smart contract, any one simple like this...

I can do it, if you would like.

Comment (jonathansmirnoff) [reply README.md]: I added the contract and also the tests. I will update this PR soon.

File: README.md (hunk 2 above)
Comment (jonathansmirnoff) [reply README.md]: I updated the this. Please check it

**Pull request merged**